packages/st2/component.makefile

```diff
@@ -67,6 +78,7 @@ wheelhouse: .stamp-wheelhouse
 bdist_wheel: .stamp-bdist_wheel
 .stamp-bdist_wheel: | populate_version requirements inject-deps
 	cat requirements.txt
+	$(PIP_BINARY) install wheel --upgrade
```

Comment:
This looks like a questionable part. Any additional background behind adding this line?

Reply:
Shouldn't `wheel` be already installed in building Docker image as a requirement? https://github.com/StackStorm/st2packaging-dockerfiles/pull/74

Reply:
Yes it should be, but there is an known bug where it's not picked up properly.

Reply:
Any reference?
Wondering why it's working for other OS flavor builds?

Reply:
[Here's the SO page ](https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci)that I was able to get around the missing wheel issue.